src/simulations/collect.jl

```diff
@@ -62,6 +66,10 @@ two `DataFrame`s, one for agent-level data and one for model-level data.
 * `mdata::Vector` means "model data to collect" and works exactly like `adata`.
   For the model, no aggregation is possible (nothing to aggregate over).
 
+* `mdata::Function` use a generator function that accepts `model` as input and
+  provides a `Vector`. Useful in combination with an [`ensemblerun!`](@ref) call that
+  requires a generator function.
+
```

Comment:
Does this need an example? Not sure... _I_ can understand it.

Reply:
We already have one. Have just expanded upon it a little.